.travis.yml

```diff
@@ -15,16 +15,32 @@ before_install:
   - sudo apt-get update
   - sudo apt-get install -o Dpkg::Options::="--force-confold" -y docker-engine
 
+install:
+  - git clone https://github.com/docker-library/official-images.git ~/official-images
+
+before_script:
+  - image="$(awk '$1 == "FROM" { print $2; exit }' $VERSION/onbuild/Dockerfile)${VARIANT:+-$VARIANT}"
+  - version="$(awk '$2 == "PYTHON_VERSION" { print $3; exit }' $VERSION/${VARIANT:+$VARIANT/}Dockerfile)"
+  - echo "Building image $image with Python version $version"
+
 script:
-  - ./build.sh
+  - docker build -t "$image" "$VERSION${VARIANT:+/$VARIANT}"
```

Comment:
I think this is the first time I've ever seen `:+` outside of the documentation. :-)